#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

float32 resolution

uint32 widthCells
uint32 heightCells

float32 widthM
float32 heightM

srslib_framework/Pose origin

string logicalFilename
